keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/FitnessFunction.java

```diff
@@ -24,10 +28,11 @@ public MultivariateFunction fitness() {
 
     protected void setAndCascadePoint(double[] point) {
         for (int i = 0; i < point.length; i++) {
-            latentVertices
-                    .get(i)
-                    .setAndCascade(point[i]);
+            Vertex<Double> v = latentVertices.get(i);
```

Comment:
nitpick, can we call this `vertex` not `v` to keep it consistent with the method below?